lib/kubeclient/informer.rb

```diff
@@ -70,15 +71,17 @@ def cache_key(resource)
     end
 
     def fill_cache
-      reply = @client.get_entities(nil, @resource_name, raw: true, resource_version: '0')
+      get_options = @options.merge(raw: true, resource_version: '0')
+      reply = @client.get_entities(nil, @resource_name, get_options)
       @cache = reply[:items].each_with_object({}) do |item, h|
         h[cache_key(item)] = item
       end
       @started = reply.dig(:metadata, :resourceVersion)
     end
 
     def watch_to_update_cache
-      @watcher = @client.watch_entities(@resource_name, watch: true, resource_version: @started)
+      watch_options = @options.merge(watch: true, resource_version: @started)
```

Comment:
Off-topic, but I wonder if `watch: true` does anything.  I don't see `watch_entities` looking at it.  @grosser do you remember why you wrote that in the first place?